infra/build/functions/build_and_run_coverage.py

```diff
@@ -223,14 +234,114 @@ def get_build_steps(  # pylint: disable=too-many-locals, too-many-arguments
       build_lib.http_upload_step(latest_report_info_body,
                                  latest_report_info_url,
                                  LATEST_REPORT_INFO_CONTENT_TYPE))
+
+  return build_steps
+
+
+def get_fuzz_introspector_steps(  # pylint: disable=too-many-locals, too-many-arguments, unused-argument
+    project_name, project_yaml_contents, dockerfile_lines, image_project,
+    base_images_project, config):
+  """Return build steps of fuzz introspector for project"""
```

Comment:
I think the docstring should be in descriptive style acording to the style guide: https://google.github.io/styleguide/pyguide.html#383-functions-and-methods

e.g. "Returns build step" instead of "Return build step"

Reply:
Done.